docs/tasks/configure-pod-container/memory-request-limit-2.yaml

```diff
@@ -6,16 +6,11 @@ metadata:
 spec:
   containers:
   - name: memory-demo-2-ctr
-    image: vish/stress
+    image: polinux/stress
     resources:
       requests:
         memory: "50Mi"
       limits:
         memory: "100Mi"
-    args:
-    - -mem-total
-    - 250Mi
-    - -mem-alloc-size
-    - 10Mi
-    - -mem-alloc-sleep
-    - 1s
+    command: ["stress"]
+    args: ["--vm", "1", "--vm-bytes", "250M", "--vm-hang", "1"]
```

Comment:
is `--vm-hang` a valid variable? When I looked at https://hub.docker.com/r/polinux/stress/, it wasn't listed as an option. Have you run this example to verify it works as expected?

Reply:
I did. It is documented at https://linux.die.net/man/1/stress